lib/admin.rb

```diff
@@ -0,0 +1,149 @@
+require 'date'
+
+class Admin
+  attr_reader :rooms, :reservations
+
+  def initialize
+    @rooms = Array.new(20)
```

Comment:
If you looked at Dan's solution, you already know that this block could be written as `@rooms = (1..20).to_a`

Reply:
Cool! Changed.